docs/pipelines/tasks/deploy/azure-resource-group-deployment.md

```diff
@@ -53,6 +53,39 @@ Use this task to deploy, start, stop, and delete Azure Resource Groups.
 |`deploymentOutputs`<br/>Deployment outputs|(Optional) Provide a name for the variable for the output variable which will contain the outputs section of the current deployment object in string format. You can use the **ConvertFrom-Json** PowerShell cmdlet to parse the JSON object and access the individual output values.|
 |`addSpnToEnvironment`<br/>Access service principal details in override parameters| Adds service principal ID and key of the Azure endpoint you chose to the script's execution environment. You can use these variables: **$servicePrincipalId** and **$servicePrincipalKey** in your override parameters like **-key $servicePrincipalKey**|
 
+## FAQs
```

Comment:
This is really good list. Should we add one more - that of template validation failing? we have historically seen issues where templates were failing due to some validation error. I think we added error message for that, but it is still good to add a section for that. also give a link in that section to the page which talk about ARM template schema/common errors. I think there was a link for that.

Reply:
Didn't get you, these are parameters to the task when written in YAML

Reply:
I think the further updates have messed up with the comment's position.
Basically, my point was to add one more item in the troubleshooting list.